install/index.adoc

```diff
@@ -51,6 +51,8 @@ configurations.
 xref:multi-masters-using-native-ha[Multiple Masters Using Native HA] example as
 a basis for your environment.
 
+* _Is xref:../install_config/prometheus_cluster_monitoring.adoc[cluster monitoring] required?_ The monitoring stack requires additional xref:../scaling_performance/scaling_cluster_monitoring.adoc[system resources]. Note that by default the monitoring stack is installed. See the xref:../install_config/prometheus_cluster_monitoring.adoc[cluster monitoring documentation] for more information.
```

Comment:
All three of the links in this bullet point back to the top of the Planning Your Installation topic, looks like each of them need anchors.